shoebot/util/video_export.py

```diff
@@ -0,0 +1,51 @@
+import os
+import shutil
+import argparse
+import subprocess
+
+
+def main():
+    global parser
+    parser = argparse.ArgumentParser("usage: sbot [options] inputfile.bot [args]")
+    parser.add_argument("script", help="Shoebot script to run")
+
+    parser.add_argument(
+        "-o",
+        "--outputfile",
+        dest="outputfile",
+        help="destination file (.mp4 only)",
+        metavar="FILE",
+    )
+    parser.add_argument(
+        "-f",
+        "--frames",
+        dest="framenumber",
+        default=300,
+        help="number of frames to export (default 300)",
+    )
+
+    args, extra = parser.parse_known_args()
+
+    if not args.outputfile:
+        outfile = args.script.replace(".bot", ".mp4")
+    else:
+        outfile = args.outputfile
+
+    tempdir = "tmp_export/"
+    tempfilename = "tmp_export/" + args.script.split("/")[-1].replace(".bot", ".png")
```

Comment:
For building paths pathlib.Path is good.   If not and and you need path separators, `os.sep` returns the path separator.

Reply:
i think it's saner now, but would like to know what you think